stock_picking_product_availability_inline/views/stock_move_line_views.xml

```diff
@@ -13,16 +13,12 @@
                 <field name="picking_type_warehouse_id" invisible="1" />
             </xpath>
             <xpath expr="//field[@name='product_id']" position="attributes">
-                <attribute
-                    name="context"
-                    operation="python_dict"
-                    key="sp_product_stock_inline"
-                >True</attribute>
-                <attribute
-                    name="context"
-                    operation="python_dict"
-                    key="warehouse"
-                >picking_type_warehouse_id</attribute>
+                <attribute name="context" operation="update">
```

Comment:
@sergio-teruel I didn't know this tag 'operation'. Great !

Reply:
Do you have a pointer to Odoo code because I can't see it anywhere...?

Reply:
@pedrobaeza Do you have a pointer ? (I understand it could be outside Odoo ?)

Reply:
> Ohhh!!! my god!!! I don't know what happened to me. I Will fix it

You gave me false hope :-)

Reply:
No hope for now: https://github.com/odoo/odoo/pull/26607

Talked in latest Odoo XP and still no luck.